source/features/reactions-avatars.tsx

```diff
@@ -49,31 +50,27 @@ function getParticipants(container: HTMLElement): Participant[] {
 }
 
 function add(): void {
-	for (const list of select.all('.has-reactions .comment-reactions-options:not(.rgh-reactions)')) {
-		const avatarLimit = arbitraryAvatarLimit - (list.children.length * approximateHeaderLength);
-
+	for (const list of select.all('.has-reactions:not(.rgh-reactions) .comment-reactions-options')) {
 		const participantByReaction = [...list.children as HTMLCollectionOf<HTMLElement>].map(getParticipants);
-		const flatParticipants = flatZip(participantByReaction, avatarLimit);
```

Comment:
@fregante `flatZip` is nowhere else used, should it be removed from `../libs/utils`?

Reply:
It can be removed.